pkg/monitor/cluster/clusterversions.go

```diff
@@ -38,14 +39,26 @@ func (mon *Monitor) emitClusterVersions(ctx context.Context) error {
 		availableRP = version.GitCommit
 	}
 
+	actualVersion := actualVersion(cv)
+	actualMinorVersion := ""
+	if len(actualVersion) > 0 {
+		parsedVersion, err := version.ParseVersion(actualVersion)
+		if err != nil {
+			return err
+		}
+		actualMinorVersion = fmt.Sprintf("%d.%d", parsedVersion.V[0], parsedVersion.V[1])
+	}
+
 	mon.emitGauge("cluster.versions", 1, map[string]string{
-		"actualVersion":                        actualVersion(cv),
+		"actualVersion":                        actualVersion,
 		"desiredVersion":                       desiredVersion(cv),
-		"provisionedByResourceProviderVersion": mon.oc.Properties.ProvisionedBy,              // last successful Put or Patch
-		"resourceProviderVersion":              version.GitCommit,                            // RP version currently running
-		"operatorVersion":                      operatorVersion,                              // operator version in the cluster
-		"availableVersion":                     availableVersion(cv, version.UpgradeStreams), // current available version for upgrade from stream
-		"availableRP":                          availableRP,                                  // current RP version available for document update, empty when none
+		"provisionedByResourceProviderVersion": mon.oc.Properties.ProvisionedBy,                 // last successful Put or Patch
+		"resourceProviderVersion":              version.GitCommit,                               // RP version currently running
+		"operatorVersion":                      operatorVersion,                                 // operator version in the cluster
+		"availableVersion":                     availableVersion(cv, version.UpgradeStreams),    // current available version for upgrade from stream
+		"availableRP":                          availableRP,                                     // current RP version available for document update, empty when none
+		"latestGaMinorVersion":                 version.InstallStream.Version.GetMinorVersion(), // Latest GA in ARO Minor version
```

Comment:
I'm a bit puzzled by presence of `resourceProviderVersion` and `latestGaMinorVersion` here: this metric is related to cluster. But the above demensions are related RP. Why do we have them here?

Reply:
@gvanderpotte

> resourcePRoviderVersion is the version of the RP that has provisionned the cluster (last PUCM by).

No, `resourcePRoviderVersion` is a version of the RP which sends the metric. It is static `version.GitCommit` constant.

Last PUCM'ed by is `provisionedByResourceProviderVersion`